genestack_client/scripts/genestack_user_setup.py

```diff
@@ -121,6 +125,26 @@ def input_email_and_password(host, alias=None):
     return connection, user
 
 
+def input_token(host, alias=None):
```

Comment:
It's kinda weird that `input_token()` not only gets token, but also logs in and returns connection and user.
Lines like `_, new_user = input_token(user.host, alias=user.alias)` look strange.

Reply:
Good point, it is artifact, that should be removed